### DISCUSSION TITLE
Работа через раз

### DISCUSSION THREAD

**Discussion by kyxxing** [dovecoteescapee/ByeDPIAndroid#163] [Q&A]
При запуске файла "**1_russia_blacklist_YOUTUBE**" или **"1_russia_blacklist_YOUTUBE_ALT"** YouTube не грузит видео, приходится запускать **"service_install_russia_blacklist_YOUTUBE_ALT"** и запускать **"1_russia_blacklist_YOUTUBE_ALT"**. 
В таком случае видео будет включатся.
 **Но спустя 2-3 видео goodbyedpi перестает работать**, и **приходится заново проводить данную махинацию.** Хочу узнать причины такой корявой работы и фикс

**Comment by ALEX2014-git** [comment]
Возможно ты можешь попробовать начать с написания в обсуждения правильного репозитория, а не в обсуждения программы которая не имеет никакого отношения к GoodbyeDPI
